automatic/kubescape/kubescape.nuspec

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>kubescape</id>
+    <version>2.2.5</version>
+    <packageSourceUrl>https://github.com/HollowMan6/chocolatey-packages/tree/master/automatic/kubescape</packageSourceUrl>
+    <owners>chocolatey-community,hollowman</owners>
+    <title>kubescape: Kubernetes Security Platform</title>
+    <authors>Kubescape authors</authors>
+    <projectUrl>https://www.armosec.io/kubescape/</projectUrl>
+    <iconUrl>https://github.com/HollowMan6/chocolatey-packages/raw/master/icons/kubescape.png</iconUrl>
+    <licenseUrl>https://github.com/kubescape/kubescape/blob/master/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <projectSourceUrl>https://github.com/kubescape/kubescape</projectSourceUrl>
+    <docsUrl>https://hub.armosec.io/docs</docsUrl>
+    <bugTrackerUrl>https://github.com/kubescape/kubescape/issues</bugTrackerUrl>
+    <tags>kubescape kubernetes security devops vulnerability-detection nsa best-practice mitre-attack</tags>
+    <summary>Kubescape is an open-source Kubernetes security platform for your IDE, CI/CD pipelines, and clusters.</summary>
+    <description><![CDATA[Kubescape is an open-source Kubernetes security platform.
```

Comment:
Note that this will be replaced by the contents (skipping the first 2 lines) of the README.

Reply:
Thank you for pointing this out.